js/ai/src/model.ts

```diff
@@ -314,6 +314,8 @@ export const GenerationUsageSchema = z.object({
   inputAudioFiles: z.number().optional(),
   outputAudioFiles: z.number().optional(),
   custom: z.record(z.number()).optional(),
+  thoughtsTokens: z.number().optional(),
+  cachedContentTokens: z.number().optional(),
```

Comment:
could you please also update https://github.com/firebase/genkit/blob/f2f7fe2c1e59c179d44fb37f605d03abee9dc450/js/ai/src/model.ts#L585

Reply:
I've updated the fields in the plugins: https://github.com/firebase/genkit/pull/2860/files#diff-25b40df99a738fd33726c36bbc7bf826741f3d08ce4b1a69845d33bebc90d3d1

Currently, the JS SDK only provides `CachedContentTokens`